commands/help.js

```diff
@@ -19,6 +19,10 @@ async function help (client, message) {
     'Locks the currently occupied room for exclusive use')
   msg.addField(`\`${settings.prefix}bitrate [ BITRATE_IN_KBPS ]\``,
     'Adjusts the bitrate of the currently occupied room')
+  msg.addField(`\`${settings.prefix}subscribe @user\``,
+    'Get a DM when @user starts practicing')
+  msg.addField(`\`${settings.prefix}unsubscribe @user\``,
```

Comment:
Non-blocking as I need to think about this for p!stats anyway due to recent Discord UI changes, but: this is going to ping the subscribee. Is that what we want?